### ISSUE TITLE
fix: Index.js file not found

### ISSUE THREAD

**Issue by tushariyer** [crisboarna/cfn-diagram-action#48]
**Description**
I just tried to add this to an action to lint a cloud formation template, and in the Action run the job for this diagram update fails with the error message "File not found: '/home/runner/work/_actions/crisboarna/cfn-diagram-action/v1/index.js'"

**Steps To Reproduce**

1. Create new Github Action to run on push/pr of path "'**/*.ya?ml'"
2. Create /docs/diagrams/ folder for output
3. Job step 1: actions/checkout@v3
4. Job step 2: crisboarna/cfn-diagram-action@v1 with the following 'with' params:
  a. path_input: '../../cloudformation'
  b. path_output: '../../docs/diagrams'
  c. diagram_type: 'draw.io'
  d. viewport_height: 1080
  e. viewport_width: 1920 
5. Commit & Push new action
6. See error

**Expected Behavior**
Diagram to generate off of the one cfn yaml file I have present

**Screenshots**

<img width="1074" alt="Screen Shot 2022-07-06 at 8 33 46 PM" src="https://user-images.githubusercontent.com/12782788/177665131-94305237-8640-4df9-b692-fa2f044819bc.png">
<img width="391" alt="Screen Shot 2022-07-06 at 8 33 14 PM" src="https://user-images.githubusercontent.com/12782788/177665134-5442e215-288e-4fbc-81f9-9a03d18cca3f.png">

**Additional Context**

**Comment by tushariyer** [comment]
Additional: Tried changing
`crisboarna/cfn-diagram-action@v1`
to 
`crisboarna/cfn-diagram-action@v1.0.0`

Same result

**Comment by tushariyer** [comment]
@crisboarna Is this project actively maintained?

**Comment by crisboarna** [comment]
Hi, yes it is maintained, have a busy couple of days with a deadline for work, I will look into this issue tomorrow and come back to you, thank you for trying out this action ! :)

**Comment by tushariyer** [comment]
Hey @crisboarna do you have any further idea of what to look for? I can also try and look at it and raise a PR but I'd like to know what you've tried so I don't duplicate that effort too

**Comment by crisboarna** [comment]
Hi @tushariyer ,

Unfortunately the work release extended couple of weeks beyond target and now I have a family event. I have not forgotten about this issue, just have not had the time :(

Based on error it seems the repo does not have the file in the error on the correct path, seems like there is a de sync between action declaration, code in ./dist. That's where I would start looking...

**Comment by crisboarna** [comment]
Hi @tushariyer ,

I got some time now and pushed update that should fix the wrong path to code.

**Comment by tushariyer** [comment]
Hi @crisboarna,

It fixed the path issue but I see a different one now. I've attached the photos of the GitHub action and the stack output. The only code change I made was to adjust 
`uses: crisboarna/cfn-diagram-action@v1`
to
`uses: crisboarna/cfn-diagram-action@v1.0.1`

Here is the stack output as text if it helps:
````
Run crisboarna/cfn-diagram-action@v1.0.1
[8](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:9)
node:internal/modules/cjs/loader:936
[9](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:10)
  throw err;
[10](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:11)
  ^
[11](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:12)

[12](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:13)
Error: Cannot find module 'ws'
[13](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:14)
Require stack:
[14](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:15)
- /home/runner/work/_actions/crisboarna/cfn-diagram-action/v1.0.1/dist/index.js
[15](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:16)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
[16](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:17)
    at Function.resolve (node:internal/modules/cjs/helpers:108:19)
[17](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:18)
    at Object.<anonymous> (/home/runner/work/_actions/crisboarna/cfn-diagram-action/v1.0.1/dist/index.js:22:86947)
[18](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:19)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
[19](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:20)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
[20](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:21)
    at Module.load (node:internal/modules/cjs/loader:981:32)
[21](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:22)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
[22](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:23)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
[23](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:24)
    at node:internal/main/run_main_module:17:47 {
[24](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:25)
  code: 'MODULE_NOT_FOUND',
[25](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:26)
  requireStack: [
[26](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:27)
    '/home/runner/work/_actions/crisboarna/cfn-diagram-action/v1.0.1/dist/index.js'
[27](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:28)
  ]
[28](https://github.com/iyer-cloud/cloud-infrastructure/runs/7640067384?check_suite_focus=true#step:3:29)
}
````

<img width="667" alt="Screen Shot 2022-08-02 at 9 44 37 PM" src="https://user-images.githubusercontent.com/12782788/182470116-a3647fc7-3a8c-4489-8e4f-a815bf84b0f9.png">
<img width="940" alt="Screen Shot 2022-08-02 at 9 44 47 PM" src="https://user-images.githubusercontent.com/12782788/182470126-d7ef33ff-2731-4ef0-bd0f-a7c48054a8bf.png">

**Comment by crisboarna** [comment]
Hi @tushariyer ,

I updated the action to use Docker and to have self-invoking integration tests to confirm it works. 

The `v1` tag tracks the latest v1.x.x release so you can stick with that ref to get the latest fixes.

**Comment by donfirst** [comment]
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/20359212/189367899-70f3da15-6a28-4fdc-9c8a-936494610f9a.png">

![image](https://user-images.githubusercontent.com/20359212/189368247-84a82e24-1ae0-4d34-9423-71766c732191.png)

**Comment by crisboarna** [comment]
Hi @donfirst 

Please use @v1 tag as that will track latest released version, which currently is @v1.1.0 which is moved to a `Docker` type action resolving the complicated dependency loading needed for a direct distributable file contained in the repository that would need playwright, chromium and other binary dependencies that are tricky to contain in this format.
